docs/platform-services/threat-intelligence-indicators.md

```diff
@@ -254,9 +306,25 @@ Since different sources can report different reputations, each source has a repu
 
 <img src={useBaseUrl('img/platform-services/threat-indicators-in-cloud-siem-ui.png')} alt="Threat indicators in the Cloud SIEM UI" style={{border: '1px solid gray'}} width="400" />
 
+### Custom threat intelligence sources in Cloud SIEM
+
+Previously, you could [create custom threat intelligence sources using Cloud SIEM](/docs/cse/administration/create-custom-threat-intel-source/). You can no longer create custom sources using Cloud SIEM, but you must use the Threat Intelligence tab, a collector, or the API to [ingest threat intelligence indicators](#ingest-threat-intelligence-indicators). If you have custom sources in Cloud SIEM, they will continue to be honored until the feature is deprecated at a future time.
```

Comment:
Do we talk about previously and currently in para? If so, can we mention **Currently or now** for the second line. Just to show the difference.

Reply:
Good point. I'll update the wording.